### ISSUE TITLE
Can't Add LiveJournal

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#547]
```
What's the problem?
Cannot add my LiveJournal blog; First, a window opens asking me "A username and 
password are being requested by http://www.livejournal.com. The site says: "lj" 
 I had already entered my user name and password & those are the only 
credentials I have, so I enter them again.  The Add A New Blog window responds 
with:  Well, this is embarrassing...

ScribeFire couldn't get the information it needed about your blog. Helpfully, 
your blog returned this message: <html><head><title>401 Authentication 
Failed</title></head><body> <h1>Authentication Failed</h1><hr 
/><p>Authentication failed for this AtomAPI request.</p> </body></html> 




What browser are you using?

Firefox 9.01



What version of ScribeFire are you running?

Scribefire Next 4.0

```

-----
Original issue reported on code.google.com by `TheBouti...@gmail.com` on 14 Jan 2012 at 6:37

**Comment by GoogleCodeExporter** [comment]
```
Same issue with the Scribefire 4.1 Chrome extension on OS X. It accepts the 
link to the main page of my Livejournal, identifies it as an Atom blog, pulls 
in the link of the api and then chokes on authentication.
```

-----
Original comment by `mrcha...@gmail.com` on 16 May 2012 at 5:31

**Comment by GoogleCodeExporter** [comment]
```
fixed with latest 4.3.1 version
```

-----
Original comment by `scribefi...@gmail.com` on 14 Apr 2014 at 6:49
* Changed state: **Fixed**
